#! /bin/sh /usr/share/dpatch/dpatch-run
## disable-dump.dpatch by Phillip Susi <psusi@cfl.rr.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Disableds building of e2dump since it fails to compile sometimes
## DP: because it violates the one definition rule by defining main
## DP: in e2dump.c and then linking to a library that contains defrag.obj
## DP: which has its own main.  It also doesn't seem useful these days given
## DP: the debugfs utility in e2fsprogs.

@DPATCH@
diff -urNad defrag-0.73pjm1~/GNUmakefile.in defrag-0.73pjm1/GNUmakefile.in
--- defrag-0.73pjm1~/GNUmakefile.in	2008-10-24 13:29:08.000000000 -0400
+++ defrag-0.73pjm1/GNUmakefile.in	2008-10-24 13:52:05.000000000 -0400
@@ -73,7 +73,6 @@
 endif
 ifdef USE_ext2
 PROGS += e2defrag
-PROGS += e2dump
 PROGS += e2defrag.static
 mansymlinks += e2defrag
 filesystems += ext2
